File: Computational Mechanics/2 (C++)/2.1 (Getting Started)/2.1.4 (Conditional Statements)/nestedif.cpp
#include <iostream>

using namespace std;

int main() {

	string my_state = "California";
	
	string my_county = "Riverside County";
	
	if (my_state == "California") {
		if (my_county == "Riverside County") {
			cout << "Your county seat is Riverside." << endl;
		}
		else if (my_county == "San Bernadino County") {
			cout << "Your county seat is San Bernadino." << endl;
		}
		else {
			cout << "I'm only programmed for Riverside and San Bernadino Counties!" << endl;
		}//end if
	
	}
	else if (my_state == "New Mexico") {
		if (my_county == "Los Alamos County") {
			cout << "Your county seat is Los Alamos." << endl;
		}
		else {
			cout << "I'm only programmed for Los Alamos County!" << endl;
		}//end if
	}
	else if (my_state == "Florida")
		if (my_county == "Miami-Dade County") {
			cout << "Your county seat is Miami." << endl;
		}
		else {
			cout << "I'm only programmed for Miami-Dade County!" << endl;
		}//end if
	
	else {
		cout << "I'm not programmed to tell you your county seat!" << endl;
	}//end if




	return 0;

}//end main

File: Computational Mechanics/3 (Python)/2.1 (Getting Started)/2.1.4 (Conditional Statements)/nestedif.py
my_state = "California"
my_county = 'Riverside County'

if my_state == "California":
	if my_county == "Riverside County":
		print("Your county seat is Riverside.")
	elif my_county == "San Bernadino County":
		print("Your county seat is San Bernadino.")
	else:
		print("I'm only programmed for Riverside and San Bernadino Counties!")
	#end if
elif my_state == "New Mexico":
	if my_county == "Los Alamos County":
		print("Your county seat is Los Alamos.")
	else:
		print("I'm only programmed for Los Alamos County!")
	#end if
elif my_state == "Florida":
	if my_county == "Miami-Dade County":
		print("Your county seat is Miami.")
	else:
		print("I'm only programmed for Miami-Dade County!")
	#end if
else:
	print("I'm not programmed to tell you your county seat!")
#end if
